pkgs/top-level/release.nix

```diff
@@ -8,7 +8,16 @@
 
    $ nix-build pkgs/top-level/release.nix -A coreutils.x86_64-linux
 */
-{ nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; revision = "0000000000000000000000000000000000000000"; }
+{ nixpkgs ? {
+  outPath = (import ../../lib).cleanSource ../..;
+  revCount = 1234;
+  shortRev = "abcdef";
+  revision = "0000000000000000000000000000000000000000";
+  legacyPackages.${system} = import ../.. {
+    inherit system;
+  };
+}
```

Comment:
This change doesn't seem to be necessary anymore (I see you used it only for an intermediate commit)

Reply:
It's necessary to evaluate with the `nix-build` tool. I added `legacyPackages` to help retain compatibility.

Reply:
Not sure what you mean, I just tested `nix-build pkgs/top-level/release.nix -A manual` and it works fine without this change (removing the `legacyPackages`). It doesn't appear to be used anywhere

Reply:
Also, I'm pretty sure Hydra uses `nix-build` instead of Flakes.

Reply:
> Not sure what you mean, I just tested `nix-build pkgs/top-level/release.nix -A manual` and it works fine without this change (removing the `legacyPackages`). It doesn't appear to be used anywhere

Huh, must've been a leftover change when I was working on trying to get things working.

Reply:
Oh wait, it is necessary. `release-lib` takes `packageSet` which is using `legacyPackages`.

Reply:
So what fails without it exactly? Because neither the above command nor `nix flake check` fail for me

Reply:
I have no idea. It might not be necessary? Kinda hard to tell with how `release-lib` and `release` works. I'll remove it and see what happens. Could be like what I mentioned earlier, leftovers from trying to fix stuff.

Reply:
Removed the change.